[IMP] Inventory: updated for v16 (#4379)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst

   :alt: Activate multi-step routes and storage locations in inventory settings.

Next, the warehouse needs to be configured for three step deliveries. To do this, go to

> :menuselection:`Inventory --> Configuration --> Warehouses`, and click on the warehouse to edit.
> Then, select :guilabel:`Pack goods, send goods in output and then deliver (3 steps)` for
> :guilabel:`Outgoing Shipments`.

Reviewer: ```suggestion
:menuselection:`Inventory app --> Configuration --> Warehouses`, and click on the warehouse to
edit. Then, select :guilabel:`Pack goods, send goods in output and then deliver (3 steps)` for
:guilabel:`Outgoing Shipments`.
```

.. image:: delivery_three_steps/three-step-warehouse-config.png
   :align: center
   :alt: Set outgoing shipment option to deliver in three steps.

> Activating three-step receipts and deliveries will create two new internal locations: a
> *Packing Zone* (WH/Packing Zone), and *Output* (WH/Output). To rename these locations, go to

Reviewer: ```suggestion
Activating three-step receipts and deliveries creates two new internal locations: a *Packing Zone*
(WH/Packing Zone), and *Output* (WH/Output). To rename these locations, go to
```
remove "will" - present tense

:menuselection:`Configuration --> Locations`, click on the :guilabel:`Location` to change, and
update the name.

> Process a delivery order in three steps (pick + pack + ship)

Author: I changed this to "Deliver in three steps (pick + pack + ship)" because that is how it appears in Odoo under the routes section. So I kept the '+'s' since that also is how it appears (this way, for people following along and looking around it'll be a perfect match).

============================================================

> 

Reviewer: Might be nice to add a quick 1-2 sentence intro into this section, briefly elaborating on why three-step process could be beneficial in some instances - just a suggestion.

Author: I put this part at the very top of the document- so the explanation of why it would be a good idea is what opens the doc. I am not sure it needs to be moved or placed again here. Or if it is not explained enough in the beginning, I can add to it as well.

Create a sales order
--------------------

In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`New`. Select a
:guilabel:`Customer`, add a storable :guilabel:`Product`, and click :guilabel:`Confirm`.

> A :guilabel:`Delivery` smart button will appear in the top right. Clicking on it will show the
> picking order, packing order, and the delivery order, which are all associated with the sales order.

Reviewer: ```suggestion
A :guilabel:`Delivery` smart button appears in the top right of the quotation form. Clicking it
shows the picking order, packing order, and the delivery order associated with the sales order.
```
present tense

   :alt: Activate multi-step routes and storage locations in inventory settings.

Next, the warehouse needs to be configured for three step deliveries. To do this, go to

> :menuselection:`Inventory app --> Configuration --> Warehouses`, and click on the warehouse to edit.

Reviewer 2: I feel that **warehouse** may need a guilabel since it is part of the active instruction
```suggestion
:menuselection:`Inventory app --> Configuration --> Warehouses`, and click on the :guilabel:`warehouse` to edit.
```
